api/src/main/java/org/openmrs/util/CurrentUsers.java

```diff
@@ -0,0 +1,67 @@
+package org.openmrs.util;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import javax.servlet.http.HttpSession;
```

Comment:
Hi @brandones  so am adding the view users functionality in core as suggested by @djazayeri here https://talk.openmrs.org/t/view-logged-in-users-is-empty/19348/3,just  a qucik question,why not have HttpSession and ServletContext implemented at core?

Reply:
Sorry, I would not be the person to answer that question. @dkayiwa or @mseaton , maybe?

Reply:
@HerbertYiga in core, they are not in the `api` but `web` module.

Reply:
@dkayiwa  does this mean it will be a nice idea to add this functionality under the web module ?

Reply:
True dat!

Reply:
@dkayiwa awesome, let me go on and do that

Reply:
Remember to test and confirm that it all works.